### ISSUE TITLE
easymotion key bind is not working

### ISSUE THREAD

**Issue by imSafdarAwan** [VSCodeVim/Vim#6828]
2 days ago the easymotion keybind stoped working <leader><leader>/ is the default keybind but leader key works for only once not twice.

**Comment by tuanna-hsp** [comment]
Same for me

**Comment by yanke1311** [comment]
same for me too.

**Comment by tuanna-hsp** [comment]
<img width="663" alt="Screen Shot 2021-07-05 at 16 43 19" src="https://user-images.githubusercontent.com/14136939/124464317-05c99d80-ddcf-11eb-9c58-0cb8a65f8de7.png">
quick workaround: install the previous release.

**Comment by RononDex** [comment]
same here

**Comment by bryan-gc** [comment]
It's broken for me too, I am using Ubuntu btw. Idk it has anything to do with it. I think it was working fine on Windows, I don't remember.

I "fixed" it by installing the 1.21.3 version.

**Comment by wizenith** [comment]
I use WSL Ubuntu on Windows, and it probably did occurred the issue to the users like me. 
My workaround is same approach as above

**Comment by kazudeath1** [comment]
same

**Comment by RononDex** [comment]
@wizenith it certainly has nothing to do with WSL or Windows, I have the same problem on native windows and native linux

**Comment by wdcew** [comment]
me too, I am using macOS 10.15

**Comment by k1832** [comment]
It occurred to me. I'm using 'space' as a leader. But when I type 2 spaces, the navigation bar shows one <leader> and my cursor moves to the right.
After I set <leader> to default, which is '\' , it worked correctly.
I think it has nothing to do with OS.

**Comment by wizenith** [comment]
> @wizenith it certainly has nothing to do with WSL or Windows, I have the same problem on native windows and native linux

I see, thanks for providing a verified info of the issue.

**Comment by J-Fields** [comment]
Duplicate of #6821

**Comment by J-Fields** [comment]
Sorry for the regression! This is fixed on 1.21.5, which is now available - please upgrade.

**Comment by imSafdarAwan** [comment]
Solved
